### PR TITLE
Move batch ~start commands under --query for workflow

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

batch terminate/cancel/signal now live here

`tctl workflow cancel --query ...`
`tctl workflow signal --query ...`
`tctl workflow terminate --query ...`

## Why?

discoverability

## Checklist

1. Closes 

2. How was this tested:

tests  

3. Any docs updates needed?

### CODE REVIEW

File: cli/workflow.go [hunk 1]
```diff
@@ -106,8 +106,23 @@ func newWorkflowCommands() []*cli.Command {
 		},
 		{
 			Name:  "signal",
-			Usage: "Signal a Workflow Execution",
-			Flags: append(flagsForExecution, []cli.Flag{
+			Usage: "Signal Workflow Execution by Id or List Filter",
```
Comment (alexshtin) [inline cli/workflow.go:109]:
```suggestion
			Usage: "Signal Workflow Execution by Id or List Query",
```

Comment (alexshtin) [reply cli/workflow.go:109]: And down bellow.

File: cli/workflow_commands.go [hunk 2]
```diff
@@ -371,14 +371,25 @@ func printWorkflowProgress(c *cli.Context, wid, rid string, watch bool) error {
 	}
 }
 
-// TerminateWorkflow terminates a workflow execution
 func TerminateWorkflow(c *cli.Context) error {
+	if c.String(FlagQuery) != "" {
+		return BatchTerminate(c)
```
Comment (alexshtin) [inline cli/workflow_commands.go:376]:
```suggestion
		return batchTerminate(c)
```
?

**Review by alexshtin — APPROVED** [review]

File: cli/workflow.go (hunk 1 above)
Comment (feedmeapples) [reply cli/workflow.go:109]: this is the new term from the docs, however we also dont want to rename the flag yet due to possible upcoming changes

File: cli/workflow_commands.go (hunk 2 above)
Comment (feedmeapples) [reply cli/workflow_commands.go:376]: ? it is uppercase from `batch_commands.go`

**Pull request merged**